packages/@okta/vuepress-theme-prose/assets/css/layouts/_authorizationLayout.scss

```diff
@@ -6,6 +6,10 @@
   flex-direction: column;
   justify-content: space-between;
 
+  .mr-15 {
```

Comment:
Can we use a more descriptive name for the class? Classes should be normally named after the element it represents, rather than what style it will apply. If the margin ever changes to something else (e.g. 20px) this class name won't make sense anymore.

Reply:
in css quite common case using a bunch of auto-generated classes for margins, paddings etc. In my case, I have only one class but I tried to follow the same idea, this class was declared on the global level because I expected that it could be used in different places of the signup page where concrete 15 margin needed. Perfectly we need to have such auto-generated classes for padding, margins, and somewhere I've seen such, we can try to use that file, what do you think?

Reply:
I've had issues with utility classes conflicting with other classes, and then having specificity resolve conflicts, which can lead to unintended side-effects in the future when the class or DOM structure changes. It works well when it's defined as a part of a broad utility styling library like Bootstrap, but IMO it's not worth trying to make it up on one's own.